components/fxa-client/examples/devices_api.rs

```diff
@@ -104,6 +104,7 @@ fn main() -> Result<(), failure::Error> {
                             };
                             webbrowser::open(&tab.url).unwrap();
                         }
+                        _ => {} // Ignore
```

Comment:
@grigoryk This unrelated change means that `FirefoxAccount.pollDeviceCommands` (which returns `Array<AccountEvent>`) could theoretically (but will never) return other types of account events such as `ProfileUpdated`. Should we rename this method to `pollAccountEvents` to make it sound more correct?